packages/db/src/query/ir.ts

```diff
@@ -128,3 +130,48 @@ export class Aggregate<T = any> extends BaseExpression<T> {
     super()
   }
 }
+
+/**
+ * Helper functions for working with Where clauses
+ */
+
+/**
+ * Extract the expression from a Where clause
+ */
+export function getWhereExpression(where: Where): BasicExpression<boolean> {
+  return typeof where === `object` && `expression` in where
+    ? where.expression
+    : where
+}
+
+/**
+ * Extract the expression from a HAVING clause
+ * HAVING clauses can contain aggregates, unlike regular WHERE clauses
+ */
+export function getHavingExpression(
+  having: Where
+): BasicExpression | Aggregate {
+  return typeof having === `object` && `expression` in having
```

Comment:
This function seems to only differ from `getWhereExpression` by the return type. But the implementation is the same. Should we unify them?